backend/routes/index.js

```diff
@@ -243,6 +243,8 @@ module.exports = function() {
           errors.push("Error with processing articles/items")
         }
       }
+      myArticles.sort((a, b) => new Date(b.uploadDate) - new Date(a.uploadDate))
+     allArticles.sort((a, b) => new Date(b.uploadDate) - new Date(a.uploadDate))
```

Comment:
Can we do this sort with a Compass Sort query ? So it happens in the db instead of in memory on the server?

I mean this is fine I guess but later on we might want to only return top-n and stuff. 

This is fine though.